Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -71,6 +71,11 @@ async function run() {
 
         var endPoint = await azureStackUtility.initializeAzureRMEndpointData(connectedServiceName);
 
+        if(slotName && slotName.toLowerCase() === 'production') {
```

Comment:
As discussed offline, Can you please handle similar case for Slot Swap action  where source or target slot is given as 'Production'

Reply:
Addressed